api/store/mongo/migrations/migration_10.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 
 	"github.com/sirupsen/logrus"
+
```

Comment:
This seems to be an unrelated change. Please drop it.

Reply:
This behavior is due to IntelliJ auto import. I did not note that.
Fixing it now

Reply:
We try to keep the changes at a minimum, thanks for reviewing it.